tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "$schema": "https://json.schemastore.org/tsconfig",
+  "extends": "./tsconfig.base.json",
+  "compilerOptions": {
+    "checkJs": true,
+    "target": "esnext"
+  },
+  "include": [".eslintrc.cjs"],
```

Comment:
Only use required file extensions for type checking in the project if I remove these lines typescript is not going to check any non ts file in the project